dashboard/app/views/levels/editors/_blockly.html.haml

```diff
@@ -1,6 +1,6 @@
 - content_for(:head) do
-  = javascript_include_tag 'js/blockly'
-  - if @level.game && @level.game.app
+  - if @level.is_a?(Blockly) && !@level.game.uses_droplet?
+    = javascript_include_tag 'js/blockly'
     = javascript_include_tag "js/en_us/blockly_locale"
     = javascript_include_tag "js/en_us/common_locale"
     = javascript_include_tag "js/en_us/#{@level.game.app}_locale"
```

Comment:
All have `game.app` for me locally:
```ruby
irb(main):001:0> Blockly.all.all?{ |l| l.game.app }
=> true
```
FWIW it looks like the only `Level` objects that don't have an associated `Game` are `FrequencyAnalysis`.  I wonder if we should fix this and make `game_id` a non-nullable column?
```ruby
irb(main):002:0> Level.where(game_id: nil).pluck(:type).uniq
=> ["FrequencyAnalysis"]
```